conf/schemas.go

```diff
@@ -116,6 +117,19 @@ func ReadSchemas(file string) (Schemas, error) {
 		}
 		schema.Priority = int64(p)<<32 - int64(i) // to sort records with same priority by position in file
 
+		reorderBufferStr := sec.ValueOf("reorderBuffer")
+		if len(reorderBufferStr) > 0 {
+			reorderWindow, err := strconv.ParseUint(reorderBufferStr, 10, 32)
```

Comment:
If we'd add options to the reorderBuffer wouldn't they most likely be appended to the current one, in a similar style like the schema string?
F.e. previously, when the reorder buffer still had the `flushMin`, the setting was like `reorderBuffer=<reorder window>,<flush min>`. I'm fine changing it to reorder window if you prefer, but that means that if we later add more configs to the reorder buffer that would probably either have to be renamed or we'd need to add them as additional separate parameters.